### PR TITLE
Smat-120: logs review

### PR DESCRIPTION
**Pull request by pavel-moskovkin**
- reviewed all middleware log messages 
- add some extra log fields inside components
- get logger instance from context once and use it inside functions

### CODE REVIEW

File: pkg/middlewares/replicate/replicate.go [hunk 1]
```diff
@@ -38,13 +39,13 @@ type replicate struct {
 
 // New creates a new http handler.
 func New(ctx context.Context, next http.Handler, config *runtime.MiddlewareInfo, middlewareName string) http.Handler {
-	ctx = middlewares.GetLoggerCtx(ctx, middlewareName, typeName)
-	log.FromContext(ctx).Debug("Creating middleware")
+	ctx = middlewares.GetLoggerCtx(ctx, middlewareName, middlewareType)
+	logger := log.FromContext(ctx)
 
+	logger.Debugf("Creating middleware %v", middlewareName)
```
Comment (raproid) [inline pkg/middlewares/replicate/replicate.go]: inconsistent with the other messages format. Suggestion: start with the lower case if possible, i.e. "creating middleware %v".

File: pkg/middlewares/replicate/replicate.go [hunk 2]
```diff
@@ -189,21 +195,20 @@ func (r *replicate) connectProducer(ctx context.Context, config *runtime.Middlew
...
 	if err != nil {
-		log.FromContext(middlewares.GetLoggerCtx(ctx, middlewareName, typeName)).
-			Warn(strings.Join([]string{"Replicate: failed to start sending alive messages", err.Error()}, ": "))
+		logger.Warnf("failed to start sending alive messages: %v", err)
 	}
 }
 
 func sendEvent(ctx context.Context, producer producer.Producer, event producer.Event, name string) {
-	logger := log.FromContext(middlewares.GetLoggerCtx(ctx, name, typeName))
+	logger := log.FromContext(middlewares.GetLoggerCtx(ctx, name, middlewareType))
```
Comment (bigbes) [inline pkg/middlewares/replicate/replicate.go]: sendEvent уже должен обладать middlewareName/middlewareType? которая приходит к нему из функции выше.

File: pkg/middlewares/replicate/replicate.go (hunk 1 above)
Comment (pavel-moskovkin) [reply pkg/middlewares/replicate/replicate.go]: fixed

File: pkg/middlewares/replicate/replicate.go (hunk 2 above)
Comment (pavel-moskovkin) [reply pkg/middlewares/replicate/replicate.go]: good point, fixed

**Review by raproid — APPROVED** [review]

**Review by megge-dream — APPROVED** [review]

**Review by bigbes — APPROVED** [review]

**Pull request merged**